### ISSUE TITLE
A11y: Add "Skip Link" for the first TAB

### ISSUE THREAD

**Issue by rvveber** [suitenumerique/docs#1621]
First TAB opens a small popup "Skip to content"
He shows it at the beginning: https://youtu.be/cOmehxAU_4s?t=38

Useful to jump directly to content, we even have `#mainContent` in layout, so it should be easy to implement universally and it helps to not need to press TAB multiple times until you finally arrive at the main content.
